decidim-proposals/spec/features/proposals_spec.rb

```diff
@@ -98,6 +126,34 @@
       expect(page).to have_content(proposal.author.name)
     end
 
+    context "when scoped_proposals setting is enabled" do
+      let!(:proposal) { create(:proposal, feature: feature, scope: scope) }
+
+      before do
+        feature.update_attributes(settings: { scoped_proposals_enabled: true})
```

Comment:
Since Hound doesn't work anymore: Missing space between } and ) (try running rubocop -a)